docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2144,6 +2152,8 @@ function addMedia() {
       forceScreenShareViewLayout(meeting);
     }
 
+    remoteVideoResolutionCheckInterval();
```

Comment:
`remoteVideoResolutionCheckInterval()` in the current implementation only works for transcoded meetings, so we should call it here only if `!isMultistream`, or you can make it work for multistream too

Reply:
Removed the function call for multistream meetings as of now. For multistream, we can take this as a separate jira as it will require some logic to add resolution checker for each video pane.